assignments/objects.js

```diff
@@ -20,25 +20,72 @@ const example = {
 // Write your intern objects here:
 
 
+const intern1 = {
+  "id": 1,
+  "name": "Mitzi, F",
+  "email": "mmelloy0@psu.edu", 
+  "gender": "F",
+}
+
+const intern2 = {
+  "id": 2,
+  "name": "Kennan,M", 
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  speak: function() {
+    return "Hello my name is Kennan";
+  }
+}
+
+const intern3 = {
+  "id": 3,
+  "name": "Keven,M",
+  "email": "mmelloy0@kmummery2@wikimedia.org", 
+  "gender": "M",
+}
+
+const intern4 = {
+  "id": 4,
+  "name": "Gannie,M",
+  "email": "gmartinson3@illinois.edu",
+  "gender": "F",
+}
+
+const intern5 = {
+  "id": 5,
+  "name": "Antonietta,F",
+  "email": "adaine5@samsung.com", 
+  "gender": "F",
+  multiplyNums: function(num1, num2) {
+    return "I can multiply numbers  " + num1 + num2;
+  }
```

Comment:
This method does not actually multiply nor even add numbers. What it does do is place the two numbers to the right of the string `"I can multiply numbers  "`.

Sample output. given `num1 === 3` and `num2 === 4`: 
```
I can multiply numbers  34
```

Two ways to multiply numbers before placing them in a string:
• Place the numbers and the mathemtical operator inside parentheses: `return "I can multiply numbers" + (num1 * num2);`
• Use a [template string literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals): `` return `I can multiply numbers ${num1 * num2}` ``